etc/ReplayOfflineConfiguration.py

```diff
@@ -225,12 +226,16 @@
 #############################
 addExpressConfig(tier0Config, "Express",
                  scenario=ppScenario,
+                 diskNode="T2_CH_CERN",
                  data_tiers=["FEVT"],
                  write_dqm=True,
                  alca_producers=["SiStripPCLHistos", "SiStripCalZeroBias", "SiStripCalMinBias", "SiStripCalMinBiasAAG",
                                  "TkAlMinBias", "LumiPixelsMinBias", "SiPixelCalZeroBias",
                                  "PromptCalibProd", "PromptCalibProdSiStrip", "PromptCalibProdSiPixelAli",
-                                 "PromptCalibProdSiStripGains", "PromptCalibProdSiStripGainsAAG", "PromptCalibProdSiPixel"
+                                 "PromptCalibProdSiStripGains", "PromptCalibProdSiStripGainsAAG",
+                                 "SiPixelCalSingleMuon", 
```

Comment:
this would be better put above among the `ALCARECO` producers than here among the `ALCAPROMPT` producers.

Reply:
ok doing it!